### PR TITLE
Disable scrollbars on ScrollViews before screenshot

### PR DESCRIPTION
**Pull request by hamorillo**
### :pushpin: References
* **Issue:** #94 
* **Related pull-requests:** #99 

### :tophat: What is the goal?

Remove scrollbar and over scroll mode on ScrollViews before make the screenshot.

### How is it being implemented?

Using the previous added method for search children views, I look for `ScrollView` and `HorizontalScrollView` and disable the scroll bars.

### How can it be tested?
Some test were implemented in project `android-consumer`

### CODE REVIEW

**Comment by hamorillo** [comment]
@pedrovgs I made some changes for avoid problems with scrollbars in ScrollViews, If you think it could be usefull just tell me if can improve something in the PR. 

I was thinking about made the same with RecyclerViews but maybe we can discuss how we could do it.

**Comment by pedrovgs** [comment]
This is just great @hamorillo thank you so much for this PR. I couldn't take a look at the code yesterday but don't worry. Next week I'll spend some time working on this library and I'll review this PR so we can release a new version including your feature 😃

File: shot-android/src/main/java/com/karumi/shot/ScreenshotTest.kt [hunk 1]
```diff
@@ -131,9 +133,26 @@ interface ScreenshotTest {
...
     }
 
+    private fun hideScrollViewBars(view: View) {
+        view.childrenViews<ScrollView>().forEach {
+            hideViewBars(it)
+        }
+
+        view.childrenViews<HorizontalScrollView>().forEach {
+            hideViewBars(it)
+        }
```
Comment (fmontesino) [inline shot-android/src/main/java/com/karumi/shot/ScreenshotTest.kt:147]: I think this should cover as well all of `NestedScrollView` childrenViews

File: shot-android/src/main/java/com/karumi/shot/ScreenshotTest.kt [hunk 2]
```diff
@@ -131,9 +133,26 @@ interface ScreenshotTest {
...
+    }
+
+    private fun hideViewBars(it: View) {
+        it.isHorizontalScrollBarEnabled = false
+        it.isVerticalScrollBarEnabled = false
```
Comment (fmontesino) [inline shot-android/src/main/java/com/karumi/shot/ScreenshotTest.kt:152]: I've found that adding `it.scrollBarSize = 0` helps to completely disable them

File: shot-android/src/main/java/com/karumi/shot/ScreenshotTest.kt (hunk 1 above)
Comment (pedrovgs) [reply shot-android/src/main/java/com/karumi/shot/ScreenshotTest.kt:147]: Hi @fmontesino, I think we can add this in a different PR.

File: shot-android/src/main/java/com/karumi/shot/ScreenshotTest.kt (hunk 2 above)
Comment (pedrovgs) [reply shot-android/src/main/java/com/karumi/shot/ScreenshotTest.kt:152]: If this implementation works, it is ok for me right now.

**Comment by pedrovgs** [comment]
Thanks for this awesome PR @hamorillo 😃

**Pull request merged**